src/types.ts

```diff
@@ -215,6 +221,7 @@ export interface JobWithMetadata<T = object> extends Job<T> {
 
 export interface JobInsert<T = object> {
   id?: string;
+  name: string;
```

Comment:
This `name` property is not used, and causes type errors. Queue name is passed on a separate parameter:
https://github.com/timgit/pg-boss/blob/2c9009fb00a4c28e11cdf724c3d772c143ef59e5/src/manager.ts#L357